services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import User, Order, MovieSession, Ticket
+
+
+def create_order(
+    tickets: list[dict], username: str, date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The transaction should be "Atomicy" - remove unnecessary operations from the context manager

Reply:
what about get_user_model method method ?

Reply:
Missed one, fixed.